### ISSUE TITLE
Thanks for posts

### ISSUE THREAD

**Issue by marcovo** [marcovo/phpbb_hidebbcode#7]
https://www.phpbb.com/community/viewtopic.php?p=13917886#p13917886

Add an optional link with the Thanks for Posts extension, also see https://www.phpbb.com/community/viewtopic.php?f=456&t=2259046&hilit=thank+for+post .

**Comment by marcovo** [comment]
Compatibility implemented in this commit: https://github.com/marcovo/phpbb_hidebbcode/commit/52ed482fd83388c73bc8ffdaf96f32644aa7b423

Still needs elaborate testing.

**Comment by marcovo** [comment]
Since commit https://github.com/marcovo/phpbb_hidebbcode/commit/101bf13f214a9ea45cd8efecee1fcefe4f5ddf06 the texts have been edited, to correctly display why messages have been hidden, and how you can reveal them (by replying, by thanking, or both).

**Comment by Galixte** [comment]
Hi @marcovo,

we have to apply those edits in this PR: https://github.com/rxu/thanks_for_posts/pull/45 to obtain the “Both” choice ?

**Comment by marcovo** [comment]
That's correct. For testing purposes, you can use the more compact edits in the first commit https://github.com/marcovo/thanks_for_posts/commit/898e5b5b92217f271888a7bea852fb4dd64c7931 . The more comprehensive second commit was made to comply to the coding guidelines. So both commits serve the same functionality, but in different forms.

**Comment by Galixte** [comment]
thnaks ;=)
